dex/asset.go

```diff
@@ -9,14 +9,15 @@ import (
 	"time"
 )
 
-type Error string
+type assetError string
 
-func (err Error) Error() string { return string(err) }
+func (err assetError) Error() string { return string(err) }
 
 const (
-	UnsupportedScriptError = Error("unsupported script type")
-	LockTimeTaker          = 24 * time.Hour
-	LockTimeMaker          = 48 * time.Hour
+	UnsupportedScriptError = assetError("unsupported script type")
```

Comment:
Would be `UnsupportedScriptError = dex.ErrorKind("unsupported script type")` and no boiler plate defs above it if we run with this.